pandas/tests/test_strings.py

```diff
@@ -572,6 +572,13 @@ def test_contains(self):
         res = values.str.contains('foo', na="foo")
         assert res.loc[2] == "foo"
 
+        # category
```

Comment:
can you make a new test and parametrize with na=True & False for both object and categorical series